README.md

```diff
@@ -45,10 +48,36 @@ Supported environment variables:
 
 Where `StatsdServerName` is the hostname or address of the StatsD server, `StatsdPort` is the optional DogStatsD port number, and `Prefix` is an optional string that is prepended to all metrics.
 
-## Usage via the static DogStatsd class.
+## Usage via the DogStatsdService class or the static DogStatsd class.
 
 For usage of DogStatsD metrics, events, and Service Checks the Agent must be [running and available](https://docs.datadoghq.com/developers/dogstatsd/?tab=net#setup).
 
+Here is an example to submit a COUNT metric with `DogStatsdService` and `DogStatsd`.
+```csharp
+// The code is located under the StatsdClient namespace
+using StatsdClient;
+
+// ...
+
+var dogstatsdConfig = new StatsdConfig
+{
+    StatsdServerName = "127.0.0.1",
+    StatsdPort = 8125,
+};
+
+// Using DogStatsdService instance
+using (var service = new DogStatsdService())
+{
+    service.Configure(dogstatsdConfig);
+    service.Increment("example_metric.increment", tags: new[]{"environment:dev"});
+}  // Flush all metrics not yet sent
+
+// Or using DogStatsd static class
```

Comment:
Maybe two separate snippets (one for the service instance, one for the static class), so that there's less room for confusion?

Reply:
Also, should we include some examples for one or two additional metric types? gauges, histograms and distributions, for example? As we as a comment stating there are many other data types.